avocado/etc/avocado/avocado.conf

```diff
@@ -29,8 +29,12 @@ per_test = False
 [sysinfo.collectibles]
```

Comment:
There is an open PR that is removing all templates files. In order to avoid duplicated help_msgs here and during the registration, IMO we should remove the changes here on this file.

Reply:
Sure, will remove changes to conf file